### PR TITLE
Issue 40 upload venue banner new

### PR DESCRIPTION
**Pull request by irmanherdiana**
Fixed pull request to staging

### CODE REVIEW

**Review by JuniYadi — CHANGES_REQUESTED** [review]
1. Error ketika upload: ValidationException: One of the required keys was not given a value
2. body value nya harus nya bukan profilePicture, tapi banner

Screenshoot terlampir

Screenshot_424

**Comment by irmanherdiana** [comment]
changes request is done, please check.
image

File: controller/profileVenueController.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import * as AWS from 'aws-sdk';
 import { v4 as uuidv4 } from 'uuid';
 import * as Multer from 'multer';
 import * as fs from 'fs';
+import { Console } from "console";
```
Comment (JuniYadi) [inline controller/profileVenueController.ts]: ini di hapus aja kalau ga di pakai

**Review by JuniYadi — CHANGES_REQUESTED** [review]
Update 1 lagi ya sesuai comment

File: controller/profileVenueController.ts (hunk 1 above)
Comment (irmanherdiana) [reply controller/profileVenueController.ts]: > ini di hapus aja kalau ga di pakai

Kok ada itu ya? persaaan gak nambahin itu. apa otomatis ya karena sempat coba2 pakai console.log.

Comment (JuniYadi) [reply controller/profileVenueController.ts]: bisa jadi

**Review by JuniYadi — APPROVED** [review]
LGTM

**Pull request merged**